pom.xml

```diff
@@ -125,9 +125,9 @@
             These dependencies should be aligned with the ones from the WildFly version we support
             See http://search.maven.org/#search|gav|1|g%3A"org.wildfly"%20AND%20a%3A"wildfly-parent"
         -->
-        <classmate.version>1.3.1</classmate.version>
-        <jsoup.version>1.8.3</jsoup.version>
-        <joda-time.version>2.9.5</joda-time.version>
+        <classmate.version>1.3.3</classmate.version>
+        <jsoup.version>1.10.3</jsoup.version>
+        <joda-time.version>2.9.9</joda-time.version>
```

Comment:
As for these ones, they are aligned on WF 10.1 versions. We will update them once we officially switch to WF 11. See the comment above this group of versions :)

Reply:
how did I missed that ?! :) force-pushed the update without these